desktop/apps/consignments2/components/submission_flow/index.js

```diff
@@ -28,28 +34,37 @@ const mapStateToProps = (state) => {
   const {
     submissionFlow: {
       currentStep,
+      isMobile,
       steps
     }
   } = state
 
+  const describeWorkComponent = isMobile ? DescribeWorkMobile : DescribeWorkDesktop
```

Comment:
I decided to break out the `Mobile` and `Desktop` versions of this form after a bit of futzing-- it's basically the difference between having a bunch of logic inside one `DescribeWork` form, and have two separate components. For the mobile version, we want to require a different set of fields (and also hide a bunch of them). The inline logic was getting pretty messy, and there actually isn't that much logic _within_ the form, except for the `locationAutocomplete` which I extracted into its own component.